jmix-translations/content/io/jmix/flowui/messages_it.properties

```diff
@@ -268,3 +268,14 @@ io.jmix.flowui.app.propertyfilter.dateinterval.model.predefined/nextMonth = Pros
 io.jmix.flowui.app.propertyfilter.dateinterval.model.predefined/lastYear = Scorso anno
 io.jmix.flowui.app.propertyfilter.dateinterval.model.predefined/thisYear = Anno corrente
 io.jmix.flowui.app.propertyfilter.dateinterval.model.predefined/nextYear = Prossimo Anno
+
+#FullCalendar
+io.jmix.fullcalendarflowui.component/months=Gennaio, Febbraio, Marzo, Aprile, Maggio, Giugno, Luglio, Agosto, Settembre, Ottobre, Novembre, Dicembre
+io.jmix.fullcalendarflowui.component/monthsShort=Gen, Feb, Mar, Apr, Mag, Giu, Lug, Ago, Set, Ott, Nov, Dic
+io.jmix.fullcalendarflowui.component/weekdays=Domenica, Lunedì, Martedì, Mercoledì, Giovedì, Venerdì, Sabato
+io.jmix.fullcalendarflowui.component/weekdaysShort=Dom, Lun, Mar, Mer, Gio, Ven, Sab
+io.jmix.fullcalendarflowui.component/weekdaysMin=Do, Lu, Ma, Me, Gi, Ve, Sa
```

Comment:
These message keys are not specific to the `flowui` module. It should be located in the `jmix-translations/content/io/jmix/fullcalendarflowui` directory.

Since there is no italian localization file for the `fillcalendarflowui` module, you will need to create one.